hathor/transaction/storage/rocksdb_storage.py

```diff
@@ -117,10 +118,12 @@ def _save_transaction(self, tx: 'BaseTransaction', *, only_metadata: bool = Fals
     def _save_static_metadata(self, tx: 'BaseTransaction') -> None:
         self._db.put((self._cf_static_meta, tx.hash), tx.static_metadata.to_bytes())
 
-    @override
-    def _get_static_metadata(self, vertex: 'BaseTransaction') -> VertexStaticMetadata | None:
+    def _set_static_metadata(self, vertex: 'BaseTransaction') -> None:
```

Comment:
Maybe rename to `_load_static_metadata()`?

Reply:
Done in https://github.com/HathorNetwork/hathor-core/pull/1014/commits/86250107215aa73a661e4437d5c9287ec2f5ba39

Reply:
Thanks!